lib/active_admin.rb

```diff
@@ -120,6 +120,8 @@ def after_load(&block)
 # Require things that don't support autoload
 require 'active_admin/engine'
 require 'active_admin/error'
+require 'active_admin/exception_app'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.